doc/devices.rst

```diff
@@ -66,3 +66,35 @@ Supported operations and observables
 .. raw:: html
 
     </div>
+
+
+**Parallel adjoint differentiation support:**
+
+The ``lightning.qubit`` device directly supports the `adjoint differentiation method <https://pennylane.ai/qml/demos/tutorial_adjoint_diff.html>`__, and enables parallelization over the requested observables (Linux/MacOS support only).
```

Comment:
A silly question: I don't remember what the issue was with getting support for Windows?

Reply:
OpenMP isn't natively supported on Windows without linking directly against the MSVC runtime.

Reply:
Or something to that effect

Reply:
Ah! Yessss this is the case :grimacing: